### PR TITLE
Events page

### PR DESCRIPTION
**Pull request by franalderete7**

### CODE REVIEW

**Comment by franalderete7** [comment]
Made progress on events page today.  The UI might look weird because I still need to work on it tomorrow and hopefully get it done before the weekend.

**Comment by franalderete7** [comment]
I have a question. Layouts like 'Schedule' on Figma need to be hardcoded for now?

**Comment by franalderete7** [comment]
I will probably create separate components for each section on events page.

**Comment by franalderete7** [comment]
Most styles are done for the events page with functionalities included. Now, I need to implement the following: Venue, Featured Speakers, Featured Sessions and Accomodation. I think they are not included in the database yet. Once sorted out, I can be done by tomorrow or the day after tomorrow.

**Comment by closerearth** [comment]
Nice - lot's of progress here.

File: components/Footer.js [hunk 1]
```diff
@@ -3,26 +3,33 @@ import Newsletter from './Newsletter'
 import Link from 'next/link'
 import api from '../utils/api'
 import { trackEvent } from './Analytics'
-import { PLATFORM_NAME, LOGO_FOOTER, LOGO_HEADER, TEAM_EMAIL, INSTAGRAM_URL, FACEBOOK_URL, TWITTER_URL, NEWSLETTER, DISCORD_URL } from '../config';
+import { PLATFORM_NAME, LOGO_FOOTER, LOGO_HEADER, TEAM_EMAIL, INSTAGRAM_URL, FACEBOOK_URL, TELEGRAM_URL, TWITTER_URL, NEWSLETTER, DISCORD_URL } from '../config';
```
Comment (closerearth) [inline components/Footer.js:6]: You already committed these changes in the other MR, they shouldn't appear in this branch.

File: pages/events/[slug]/index.js [hunk 2]
```diff
@@ -94,158 +107,572 @@ const Event = ({ event, error }) => {
...
+                        esse cillum dolore eu fugiat nulla pariatur.</p>
+                      <p className="underline text-sm">learn more</p>
+                    </div>
+                  </div>
+                  
+                  <button className="btn-primary w-52 mt-8">See more speakers</button>
+                </div>
+              </div>
+
+              <div className="flex flex-col space-y-4 md:space-y-0 md:flex-row items-start justify-start w-full md:ml-full md:mr-full pb-12 pt-12 border-b border-black" >
```
Comment (closerearth) [inline pages/events/[slug]/index.js:467]: Can we break out each of these sections to a new component please?

**Comment by closerearth** [comment]
Screen Shot 2022-03-11 at 1 43 25 PM
Layout is completely broken.

**Comment by closerearth** [comment]
I think this MR can be closed.

1. There are way too many changes for a single MR
2. It completely breaks the functionality

Please make a new MR that creates incremental progress on individual pieces. For example:
Make 1 MR to add the speakers section to the page. The code should be in its own component mostly and not break existing page. The code should be functional - meaning all buttons work etc.

**Pull request closed without merging**